spartan/aztec-network/templates/pxe.yaml

```diff
@@ -103,6 +133,10 @@ apiVersion: v1
 kind: Service
 metadata:
   name: pxe-lb
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
+    service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: "ip"
+    service.beta.kubernetes.io/aws-load-balancer-scheme: "internet-facing"
```

Comment:
since the pxe doesn't join the p2p network, it can have a load balancer when deploying public